### DISCUSSION TITLE
Glitches

### DISCUSSION THREAD

**Discussion by cinemachameleon** [radubirsan/FlashBrowser#21] [General]
I believe I've encountered a glitch. Every time I start up a flash game / animation, a strange song plays over all the other audio, not part of the flash game / animation, and it continues to play until I stop the game / animation. It only started doing this recently. I uninstalled Flash Browser and reinstalled it. It changed the song, for some reason, but the problem persists.

**Comment by radubirsan** [comment]
Can you share the links you are accessing,  I need a way to reproduce this to be able to fix this. I saw as similar complaint in a YouTube video comment.

**Comment by cinemachameleon** [comment]
Any game on the home page your Flash Browser defaults to. It doesn't seem
to matter which one. I didn't used to do it, it used to work just fine,
then one day it just started playing random music over the audio of any
game I clicked on. Then I uninstalled and reinstalled it and it changed the
music but the problem persisted.

[image: width=]
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by yamcrunchy** [comment]
bump same for me
